src/frontends/pytorch/src/frontend.cpp

```diff
@@ -42,8 +42,10 @@ std::set<std::string> get_unconverted_types_from_model(const std::shared_ptr<Mod
     std::set<std::string> unconverted_ops_types;
     for (const auto& node : model->get_ordered_ops()) {
         if (const auto& fw_node = ov::as_type_ptr<PtFrameworkNode>(node)) {
-            auto op_type = fw_node->get_decoder()->get_op_type();
-            unconverted_ops_types.insert(op_type);
+            auto attrs = fw_node->get_attrs();
+            FRONT_END_GENERAL_CHECK(attrs.find("PtTypeName") != attrs.end(),
+                                    "FrameworkNode attributes do not contain PtTypeName.");
```

Comment:
user unfriendly message error, please correct if possible. `PtTypeName` is confusing